src/modules/0.0.22/iasql_functions/index.ts

```diff
@@ -3,8 +3,10 @@ import { ModuleBase } from '../../interfaces';
 import { IasqlOperationType } from './entity';
 import {
   IasqlApply,
```

Comment:
Will the Apply And Sync functions be deprecated in this PR, or are you thinking of doing so in a follow-up PR?

Reply:
Now that you have mentioned this, there are some issues still pending in this PR:
- I cannot deprecate `apply` and `sync` until we release 0.0.22
- `install` and `upgrade` would need to be modified to use the new `commitSync`, I need to see what might have changed there.

Reply:
Should we run a full `commit` while `install` and `update`? If so, and a commit is already in process? should we wait for completion?